frontend/wallet/src/common/Time.re

```diff
@@ -0,0 +1,61 @@
+let monthOfIntExn =
+  fun
+  | 0 => "January"
+  | 1 => "Februrary"
+  | 2 => "March"
+  | 3 => "April"
+  | 4 => "May"
+  | 5 => "June"
+  | 6 => "July"
+  | 7 => "August"
+  | 8 => "September"
+  | 9 => "October"
+  | 10 => "November"
+  | 11 => "December"
+  | x => failwith(Printf.sprintf("Not a month: %d", x));
+
+let thOfDate = x =>
+  switch (x mod 10) {
+  | 1 => "st"
+  | 2 => "nd"
+  | 3 => "rd"
+  | _ => "th"
+  };
+
+let render = (~date, ~now) => {
+  let int = (f, x) => f(x) |> Js.Math.unsafe_round;
+
+  let militaryHours = int(Js.Date.getHours, date);
+  let minutes = int(Js.Date.getMinutes, date);
+
+  let nowMillis = Js.Date.getTime(now);
+  let dateMillis = Js.Date.getTime(date);
+  let epsilon = 1000. *. 60.; // 60 seconds in millis
+
+  Js.log4("Epsilon", epsilon, nowMillis, dateMillis);
+
+  if (dateMillis +. epsilon >= nowMillis) {
+    "seconds ago";
```

Comment:
I know I probably suggested this in person, but now that I'm thinking about it, if we're not guaranteed to be regularly updating these times, we should probably avoid a short term relative measurement like this and just remove this case.